src/frontend/packages/auth-provider/src/components/LocalLoginPage/LocalLoginPage.js

```diff
@@ -101,15 +106,15 @@ const LocalLoginPage = ({
         {isLogin && <LoginForm redirectTo={redirectTo} allowUsername={allowUsername} />}
         <div className={classes.switch}>
           {isSignup && (
-            <Link to="/login">
+            <Link to={'/login?' + searchStr}>
```

Comment:
I've also fixed this on the `next` branch so you will get merge conflicts. But you should be able to use what I did, because it is pretty much the same as you (add search params to login and signup screens).